File: Codechef/Practice/Good Turn.cpp
https://www.codechef.com/problems/GDTURN




#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t;
	cin >> t;
	while(t--){
	    int x,y;
	    cin >> x >> y;
	    (x + y > 6) ? cout << "YES" : cout << "NO";
	    cout << endl;
	}
	return 0;
}

File: Codechef/Practice/Good Turn.go
https://www.codechef.com/practice/course/go/PGOL05/problems/GDTURN




package main

import "fmt"

func main() {
    var t int

    fmt.Scanf("%d", &t)

    // Loop for each test case
    for t > 0 {
        var x, y int
        fmt.Scanf("%d %d", &x, &y)

        // Your code for each test case goes here
        if(x+y > 6){
            fmt.Println("YES");
        }else{
            fmt.Println("NO");
        }
        
        t--
    }
}